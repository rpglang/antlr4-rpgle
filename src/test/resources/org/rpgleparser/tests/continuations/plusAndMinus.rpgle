     D Error_12        C                   ' No longer Exists on the File - +  
     D                                      Press Enter to Continue' 