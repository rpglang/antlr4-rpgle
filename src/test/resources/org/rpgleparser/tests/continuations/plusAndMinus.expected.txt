/*===TOKENS===*/
DS_FIXED                 <D>
NAME                     <Error_12>
EXTERNAL_DESCRIPTION     < >
DATA_STRUCTURE_TYPE      < >
DEF_TYPE_C               <C >
FROM_POSITION            <       >
TO_POSITION              <       >
DATA_TYPE                < >
DECIMAL_POSITIONS        <  >
RESERVED                 < >
StringLiteralStart       <'>
StringContent            < No longer Exists on the File - >
StringContent            <Press Enter to Continue>
StringLiteralEnd         <'>
/*===TREE===*/
(r
  (statement
    (dcl_c
      D
      (ds_name Error_12)
      ' '
      ' '
      'C '
      '       '
      '       '
      ' '
      '  '
      ' '
      (literal ' ' No longer Exists on the File - ' Press Enter to Continue ')
      <EOF>
    )
  )
)
/*======*/